src/UNL/UCBCN/Manager/Controller.php

```diff
@@ -205,6 +206,26 @@ public function getCalendars() {
         return $user->getCalendars();
     }
 
+    public function hasPermission($permission_description, $calendar_id) 
+    {
+        switch ($permission_description) {
```

Comment:
I'm going to have to disagree with you...

[Tue Jun 23 21:30:32 2015] [error] [client 10.0.2.2] PHP Fatal error:  Class 'Permission' not found in /var/www/html/www/templates/default/html/manager/Calendar.tpl.php on line 31, referer: http://iimlemburg.unl.edu:8008/manager/immediate77/users/s-tlembur1/edit/